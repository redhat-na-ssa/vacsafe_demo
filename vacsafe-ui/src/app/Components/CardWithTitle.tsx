import * as React from 'react';
import { 
CardTitle,
Text, TextContent, TextVariants
} from '@patternfly/react-core';

const CardWithTitle = (props) => {
  return (
     <CardTitle>
      <Text component={TextVariants.p}>
        {props.title}
      </Text>
      <TextContent>
        <Text component={TextVariants.small}>
            {props.info}
        </Text>
      </TextContent>
     </CardTitle>
  )
}

export default CardWithTitle;