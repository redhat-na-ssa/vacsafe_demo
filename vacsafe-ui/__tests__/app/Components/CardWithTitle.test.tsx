import React from 'react';
import { shallow } from 'enzyme';
import CardWithTitle from '../../../src/app/Components/CardWithTitle';

describe('<CardWithTitle />', ()=> {
    it('should be defined', () => {
        expect(CardWithTitle).toBeDefined()
    })
    it("should render without crashing", () => {
        const wrapper = shallow(<CardWithTitle />)
        expect(wrapper)
    })
    it("should match existing snapshot", () => {
        const wrapper = shallow(<CardWithTitle />)
        expect(wrapper).toMatchSnapshot()
    })
})
