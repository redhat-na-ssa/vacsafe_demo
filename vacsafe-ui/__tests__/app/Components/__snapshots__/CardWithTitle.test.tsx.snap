// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardWithTitle /> should match existing snapshot 1`] = `
<CardTitle>
  <Text
    component="p"
  />
  <TextContent>
    <Text
      component="small"
    />
  </TextContent>
</CardTitle>
`;
